src/helpers.rs

```diff
@@ -33,20 +33,27 @@ where
 /// Return all data from `path`. Line boundaries are normalized from
 /// "\r\n" to "\n" to make sure "^" and "$" will match them. See
 /// https://github.com/rust-lang/regex/issues/244 for details.
-pub fn read_file(path: &str) -> io::Result<String> {
-    let mut file = File::open(path)?;
+#[cfg(any(feature = "html_root_url_updated", feature = "markdown_deps_updated",
```

Comment:
Hmm, is this not all the features? :-) Meaning the function could just be unconditionally included?

Reply:
Both instances of the "all the features" cfg have been removed.